packages/salesforcedx-vscode-lwc/package.json

```diff
@@ -26,8 +26,8 @@
     "@salesforce/salesforcedx-utils-vscode": "43.8.0",
     "ajv": "^6.1.1",
     "eslint": "4.16.0",
-    "eslint-plugin-lwc": "0.3.2",
-    "lwc-language-server": "1.5.1",
+    "eslint-plugin-lwc": "0.4.0",
+    "lwc-language-server": "1.6.4",
```

Comment:
are you sure you want to upgrade this also?, 1.5.1 was for 214

Reply:
you are right, I'll revert this change.